src/math/numbers.c

```diff
@@ -5,6 +5,9 @@
 // Author: Seppo Ingalsuo <seppo.ingalsuo@linux.intel.com>
 //         Liam Girdwood <liam.r.girdwood@linux.intel.com>
 //         Keyon Jie <yang.jie@linux.intel.com>
+/* Binary GCD algorithm
+ * https://en.wikipedia.org/wiki/Binary_GCD_algorithm
+ */
 
```

Comment:
It looks these lines above are added twice. They shouldn't be present in this patch.